src/pages/resultsView/survival/SurvivalChart.tsx

```diff
@@ -30,9 +31,44 @@ export interface ISurvivalChartProps {
     xLabelWithEventTooltip: string;
     xLabelWithoutEventTooltip: string;
     fileName: string;
-    showTable: boolean;
-    showLegend: boolean;
-    showDownloadButtons: boolean;
+    showTable?: boolean;
```

Comment:
@ersinciftci did you make similar changes or is it my imagination?

Reply:
Hongxin converted them to optional, so they are `true` now by default. In my version they were required and didn't have a default value. I think this is better, it makes sense to show everything by default.